autoreject/autoreject.py

```diff
@@ -515,48 +498,70 @@ def _get_epochs_interpolation(self, epochs, drop_log,
                     interp_chs_mask[
                         sorted_ch_idx_picks[n_interpolate:]] = False
 
-            fix_log[epoch_idx][interp_chs_mask] = 2
-            interp_chs = np.where(interp_chs_mask)[0]
-            interp_chs = [ch_name for idx, ch_name in enumerate(ch_names)
-                          if idx in interp_chs]
-            interp_channels.append(interp_chs)
-        return interp_channels, fix_log
+            labels[epoch_idx][interp_chs_mask] = 2
+        return labels
 
-    def _get_bad_epochs(self, bad_sensor_counts, ch_type):
-        """Get the indices of bad epochs."""
+    def _get_bad_epochs(self, bad_sensor_counts, ch_type, picks):
+        """Get the mask of bad epochs."""
         sorted_epoch_idx = np.argsort(bad_sensor_counts)[::-1]
         bad_sensor_counts = np.sort(bad_sensor_counts)[::-1]
-        n_channels = len(self.picks)
-        n_consensus = self.consensus_perc[ch_type] * n_channels
+        n_channels = len(picks)
+        n_consensus = self.consensus_[ch_type] * n_channels
+        bad_epochs = np.zeros(len(bad_sensor_counts), dtype=np.bool)
         if np.max(bad_sensor_counts) >= n_consensus:
             n_epochs_drop = np.sum(bad_sensor_counts >=
                                    n_consensus)
             bad_epochs_idx = sorted_epoch_idx[:n_epochs_drop]
-        else:
-            n_epochs_drop = 0
-            bad_epochs_idx = []
-
-        return bad_epochs_idx, sorted_epoch_idx, n_epochs_drop
+            bad_epochs[bad_epochs_idx] = True
+        return bad_epochs
 
-    def _annotate_epochs(self, threshes, epochs):
+    def _get_reject_log(self, epochs, picks):
         """Get essential annotations for epochs given thresholds."""
```

Comment:
do a search the terms "annotate" and "annotations". We use the wording "drop log" now.